app/assets/javascripts/components/articles/available_articles.jsx

```diff
@@ -64,6 +65,13 @@ const AvailableArticles = createReactClass({
     }
 
     const showAvailableArticles = elements.length > 0 || this.props.current_user.isAdvancedRole;
+    const showMyArticlesSection = this.props.current_user.isStudent;
```

Comment:
I was thinking this could be shown only once there is at least one assigned article. That way, the list of available articles is still the main focus until you've chosen an article, and then MyArticles appears basically just to visualize "where" the article you chose went.